tests/snap_test.go

```diff
@@ -26,64 +30,31 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
-func TestMatterDeviceOperations(t *testing.T) {
-	const (
-		chipAllClusterMinimalAppFile = "bin/chip-all-clusters-minimal-app-commit-1536ca2"
-		chipAllClusterMinimalAppLog  = "chip-all-clusters-minimal-app.log"
-	)
-
-	// Setup: remove exisiting log files
-	if err := os.Remove("./" + chipAllClusterMinimalAppLog); err != nil && !os.IsNotExist(err) {
-		t.Fatalf("Error deleting log file: %s\n", err)
-	}
-	if err := os.Remove("./chip-tool.log"); err != nil && !os.IsNotExist(err) {
-		t.Fatalf("Error deleting log file: %s\n", err)
-	}
-
-	// Setup: run and log chip-all-clusters-minimal-app in the background
-	logFile, err := os.Create(chipAllClusterMinimalAppLog)
-	if err != nil {
-		t.Fatalf("Error creating log file: %s\n", err)
-	}
-
-	cmd := exec.Command("./" + chipAllClusterMinimalAppFile)
-	cmd.Stdout = logFile
-	cmd.Stderr = logFile
-
-	err = cmd.Start()
-	if err != nil {
-		t.Fatalf("Error starting application: %s\n", err)
-	}
+func TestAllClustersApp(t *testing.T) {
```

Comment:
This function's purpose is to test the commission and control function of the chip-tool. AllClustersApp is only a helper snap here, rather than the main purpose. How about updating the function name to something such as `TestCommissionAndControl()`?

Reply:
I agree. However, the full test do end up having meaningful names:
```
--- PASS: TestAllClustersApp (3.34s)
    --- PASS: TestAllClustersApp/Commission (0.98s)
    --- PASS: TestAllClustersApp/Control (1.[35](https://github.com/canonical/chip-tool-snap/actions/runs/7904195317/job/21574301422?pr=45#step:5:36)s)
```
An accurate parent test function name could be `TestCommissionAndControlAllClustersApp` because it is testing commission and control of that app rather than some mocked generic target. But that is a very long name and mostly redundant when combined with subtest names.